gravitee-apim-portal-webui-next/src/components/subscription/webhook/consumer-configuration/consumer-configuration.component.ts

```diff
@@ -66,69 +62,33 @@ import { ConsumerConfigurationSslComponent } from '../consumer-configuration-ssl
   styleUrls: ['./consumer-configuration.component.scss'],
 })
 export class ConsumerConfigurationComponent implements OnInit {
+  @Output()
+  save = new EventEmitter();
+
   @Input()
-  subscriptionId!: string;
+  subscription: Subscription | undefined;
+
+  @Input()
+  consumerConfigurationFormData: ConsumerConfigurationValues | null = null;
+
+  @Output()
+  consumerConfigurationFormDataChange = new EventEmitter<ConsumerConfigurationFormData>();
 
+  error: InputSignal<boolean> = input<boolean>(false);
+
+  isUpdate = false;
   consumerConfigurationForm!: ConsumerConfigurationForm;
```

Comment:
Maybe we can directly create the form here and not in the initForm method. Wdyt ?